Source/WTF/wtf/text/StringImpl.h

```diff
@@ -780,21 +781,22 @@ inline int codePointCompare(const StringImpl* string1, const StringImpl* string2
     return codePointCompare(string1->characters16(), string1->length(), string2->characters16(), string2->length());
 }
 
-inline bool isSpaceOrNewline(UChar32 character)
+inline bool deprecatedIsSpaceOrNewline(UChar32 character)
 {
-    // Use isUnicodeCompatibleASCIIWhitespace() for all Latin-1 characters. This will include newlines, which aren't included in Unicode DirWS.
+    // Use isUnicodeCompatibleASCIIWhitespace() for all Latin-1 characters, which is incorrect as it
+    // excludes U+0085 and U+00A0.
     return isLatin1(character) ? isUnicodeCompatibleASCIIWhitespace(character) : u_charDirection(character) == U_WHITE_SPACE_NEUTRAL;
```

Comment:
Looking at https://www.unicode.org/Public/UCD/latest/ucd/extracted/DerivedBidiClass.txt which I think is what this will match, it seems this will also do the wrong thing for U+2029 and U+ 202F. Though that's due to `u_charDirection`.